build-tools/src/main/resources/hawkular-checkstyle/checkstyle.xml

```diff
@@ -54,6 +54,13 @@
       <property name="processJavadoc" value="true" />
     </module>
 
+    <module name="ImportOrder">
+        <property name="groups" value="/^javax?\./,org,com"/>
```

Comment:
Does this ensure java, javax ordering? Or should we make that more explicit (which could be more readable)?